core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadService.java

```diff
@@ -227,10 +227,6 @@ public void onDestroy() {
         }
         unregisterReceiver(cancelDownloadReceiver);
 
-        stopForeground(true);
-        NotificationManager nm = (NotificationManager) getSystemService(NOTIFICATION_SERVICE);
-        nm.cancel(R.id.notification_downloading);
```

Comment:
These seem completely unnecessary to me. The code shouldn't get here while the notification is visible, but if by some random chance it does, the notification will be closed by the OS.